src/harness/reference_models/move_list_calc/README.md

```diff
@@ -0,0 +1,62 @@
+# The WinnForum R2-SGN-24 DPA Protection Procedure Reference Implementation
+
+This folder contains Python code and test data for calculation of the DPA move-list 
+according to the R2-SGN-24 requirement in the WINNF-TS-0112 document, Version V1.4.1-r2.0 
+(January 16, 2018) [1].
+
+## Main Interface 
+
+The main routine to find which CBSD grants are on the move-list is 
+`move_list.findMoveList()`. 
+
+## Code Prerequisites
+
+Reference models:
+
+   - WInnForum ITM propagation reference model (https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/tree/master/src/harness/reference_models/propagation)   
+   - WInnForum geo modules and geo data (https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/tree/master/src/harness/reference_models/geo)
+   - WInnForum antenna gain models (https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/tree/master/src/harness/reference_models/antenna)
+
+Python software and packages:
+   - python 2.7 (https://www.python.org/download/releases/2.7/)
+   - shapely (https://pypi.python.org/pypi/Shapely)
+   - numpy (http://www.scipy.org/scipylib/download.html)
+   
+## Test Data
+
+To test the `move_list.findMoveList()` routine for a co-channel offshore DPA protection, 
+one can use the example given in the `move_list_example.py` file along with the test 
+data stored in the `\test_data` sub-folder. To run the example, use the command 
+`python move_list_example.py`.
+
+## Copyrights and Disclaimers
+
+This software was developed by employees of the National Institute of Standards and 

```

Comment:
Anirud's reply:

"""
When NIST contributed to the SAS UUT test code, we put language as “.. part of this code is developed by NIST….   US Govt. work is not subject to copyright ..” etc. You can look at the exact language in WINNF_FT_S_GRA_testcase.py. But in that file, other companies did not put any claim/copyright of their own. However , that file has Apache license agreement at the top.

So, I am not sure how it should be handled in this case (I am not a lawyer either).

thanks and regards
-Anirud
"""


If you're OK to use "Some parts of this..." rather than "This..." (as you did for the grant testcase), that addresses all of my not-a-lawyer concerns :)

Reply:
NIST requires that we use the current text as a disclaimer.  It does state that others may “use, copy, modify … provided that this notice and disclaimer of warranty appears in all copies.”